schemas/data/abandons.schema.json

```diff
@@ -5,7 +5,7 @@
     "you may not use this file except in compliance with the License. You may obtain a copy",
     "of the License at https://creativecommons.org/licenses/by/4.0/"
   ],
-  "$id": "https://ns.adobe.com/xdm/data/metrics/commerce/abandons",
+  "$id": "https://ns.adobe.com/xdm/metrics/cartAbandons",
```

Comment:
The file name is conflicting with the metric name and the metric name is conflicting with the established patterns. My suggestion:

1. rename the `$id` to `https://ns.adobe.com/xdm/data/metrics/commerce/cart-abandons`
2. rename the file to to `cart-abandons.schema.json`